recipe/meta.yaml

```diff
@@ -1,75 +1,93 @@
 {% set name = "bentoml" %}
-{% set version = "0.13.1" %}
+{% set version = "1.0.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/BentoML-{{ version }}.tar.gz
-  sha256: 2f53bd2b659e305a97f5cb46fe97f92f65416959e29199fdc55b7637bf7f247b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bentoml-{{ version }}.tar.gz
+  sha256: 6bc4a4bb8f53b6ee70d1169ee439d89680f96771fbb3f11fb5eba13f96598c88
 
 build:
   number: 0
   noarch: python
```

Comment:
```suggestion
  noarch: python
  skip: true  # [py<37]
```

Reply:
`skip: true` is not used for `noarch` packages.